.circleci/config.yml

```diff
@@ -326,6 +326,9 @@ pytorch_windows_params: &pytorch_windows_params
     cuda_version:
       type: string
       default: "10"
+    cuda_complete_version:
```

Comment:
BTW, I don't think CUDA COMPLETE VERSION is good name here. The correct names and the corresponding values:
cuda version: 10.1
cuda major version: 10
cuda complete version 10.1.249

Reply:
Your comments is valuable. In some scripts,  the name of cuda_version is used as cuda major version. Let me think if I should update them all in this PR.